_config.yml

```diff
@@ -202,6 +202,20 @@ gitment:
     # Your client secret:
     client_secret:
 
+# Your Gitalk information
+gitalk:
+    # Switch
+    enable: false
+    # Your Github ID (Github username):
+    github_id:
+    # The repo to store comments:
+    repo:
+    # Your client ID:
+    client_id:
+    # Your client secret:
+    client_secret:
+    # Distraction Free:
+    distraction_free: false
```

Comment:
Regarding your suggestion:
> Gitalk does offer more configuration options which I would be happy to pass through as variable to the config, just let me know.

I suggest we keep `enable`, `github_id`, `repo`, `client_id`, `client_secret` at the root and then we add a key `options` so users can pass all other properties. E.g:
``` yaml
enable: true
owner: LouisBarranqueiro
repo: gitalk-blog-comments
client_id: 32f32fj230d23h23jf32
client_secret: jd2083mf8-4fn-3nf3f32
options:
    distractionFreeMode: false
    language: 'fr'
    flipMoveOptions:
        staggerDelayBy: 40
```
so we can easily spread and pass this options to Gitalk. What do you think?

Reply:
I like this idea! I'll do something like that.